assets/video2x-qt6.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
.desktop file probably should go into a `resources/` or something directory. I'll see what's the best option and update it.